bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/dto/ActionDTOMapper.java

```diff
@@ -1,11 +1,10 @@
 /**
- * Copyright (c) 2016 Markus Rathgeb
+ * Copyright (c) 1997, 2016 by ProSyst Software GmbH and others.
```

Comment:
I see that comment, but my reading has been that you ask if it is okay to change the ProSyst copyright header to a neutral one.
Changing a copyright to a non neutral form from holder A to holder B is from my understanding not correct.